030-reserves-pickupnumber.patch
Patch for C4::Reserves - adding pickupnumber column to GetReserveInfo used in holds slip

Now available in template:
<<reserves.pickupnumber>>

diff --git a/C4/Reserves.pm b/C4/Reserves.pm
index 68c608d..8991491 100644
--- a/C4/Reserves.pm
+++ b/C4/Reserves.pm
@@ -1428,6 +1428,7 @@ sub GetReserveInfo {
                    reserves.biblionumber,
                    reserves.branchcode,
                    reserves.waitingdate,
+                   reserves.pickupnumber,
                    notificationdate,
                    reminderdate,
                    priority,
